pkg/cli/node.go

```diff
@@ -559,7 +591,13 @@ func runRecommissionNode(cmd *cobra.Command, args []string) error {
 
 	s := serverpb.NewStatusClient(conn)
 
-	nodeIDs, err = handleNodeDecommissionSelf(ctx, s, nodeIDs, "recommissioning")
+	r, err := s.Node(ctx, &serverpb.NodeRequest{NodeId: "local"})
```

Comment:
[nit] This block used here and above could be a tiny helper, "getLocalNodeID" perhaps?

Reply:
Done.